css/src/components/checkbox.scss

```diff
@@ -0,0 +1,113 @@
+/* stylelint-disable selector-max-specificity, max-nesting-depth, selector-no-qualifying-type */
+
+$checkbox-border-radius: $radius-sm !default;
+$checkbox-unchecked-border-color: $text !default;
+$checkbox-color: $primary !default;
+$checkbox-check-color: $primary-invert !default;
+$checkbox-check-hover-color: $text-subtle !default;
+$checkbox-size: 1.25em !default;
+$checkbox-icon-size: 0.875em !default;
+$checkbox-timing-function: $input-timing-function !default;
+$checkbox-duration: $input-transition-duration !default;
+$checkbox-spacing: $spacer-3;
+
+@mixin checkmark {
+	display: block;
+	position: absolute;
+	top: 0.2em;
```

Comment:
should the em sizes in this file map a little better to font sizes at 16px?
1px = 0.0625
2px = .125
3px = .1875 etc.

Reply:
updated